pages/price-feeds/how-pyth-works/schedule-format.mdx

```diff
@@ -0,0 +1,52 @@
+# Schedule Format
+
+This document describes the format for the `schedule` field in Pyth product metadata.
+The `schedule` field defines a product's market hours, including its timezone, weekly schedule, and any holiday exceptions.
+It instructs the `pyth-agent` when to stop publishing outside the specified hours.
```

Comment:
It can be simplified as "It serves as a reference to `pyth-agent` to stop publishing outside specific time frame.

Or 

It specified `pyth-agent` publishing time frame.

Reply:
I improved the writing.